lib/tjbot.js

```diff
@@ -806,6 +806,8 @@ TJBot.prototype.read = function() {
  * Returns the photo data in a Buffer.
  */
 TJBot.prototype._captureImage = function(filePath) {
+    this._assertCapability('see');
```

Comment:
could this just be because we're not giving that "timeout" argument?

no reason in particular for using this node package, it just seemed easier than other methods (including invoking `raspistill` ourselves).  however, if it's the wrong choice, i'm not against going back to manual invocation (in a separate PR).